server/block/skull.go

```diff
@@ -0,0 +1,112 @@
+package block
+
+import (
+	"github.com/df-mc/dragonfly/server/block/cube"
+	"github.com/df-mc/dragonfly/server/block/model"
+	"github.com/df-mc/dragonfly/server/internal/nbtconv"
+	"github.com/df-mc/dragonfly/server/item"
+	"github.com/df-mc/dragonfly/server/world"
+	"github.com/go-gl/mathgl/mgl64"
+)
+
+// TODO: Dragon Heads can be powered by redstone
+
+// Skull is a decorative block. There are six types of skulls: player, zombie, skeleton, wither skeleton, creeper,
+// and dragon.
+type Skull struct {
+	transparent
+
+	// Type is the type of the skull.
+	Type SkullType
+	// Direction is the direction the skull is facing. For skulls placed on the floor, this is cube.FaceUp.
+	Direction cube.Face
+	// Rotation is the number of rotations for skulls placed on the floor. There are a total of 16 rotations.
+	Rotation cube.Orientation
```

Comment:
I think this should use the `Attachment` type. If I'm not mistaken the placement logic of skulls (roughly) match those of signs.